### ISSUE TITLE
Plugin should require confirmation when building with plugin id "com.mattermost.plugin-starter-template"

### ISSUE THREAD

**Issue by mickmister** [mattermost/mattermost-plugin-starter-template#152]
When I use the starter template for a new project and deploy the plugin to my environment, it always deploys as `Plugin Starter Template` with id `com.mattermost.plugin-starter-template`. This is because I'm not thinking about changing the plugin id when I'm creating a POC.

I propose we make it so running the `make deploy` command (and others like it that build the plugin) would first ask the user something like:

> Are you sure you want to build the plugin with an id of "com.mattermost.plugin-starter-template"? Consider editing plugin.json to configure your project.

**Comment by mickmister** [comment]
@hanzei What are your thoughts on this?

**Comment by hanzei** [comment]
@mickmister I don't have objection to this change, but I would suggest to only log a message and don't block the flow of a user.

**Comment by haardikdharma10** [comment]
Hey @mickmister! This could be a nice addition. @hanzei has a valid point too. Can we change the Makefile wherein we can use `$(info message)` or `$(warning message)` in the deploy command, both of which won't stop the execution. If yes, which one do you think will be better to use?

**Comment by mickmister** [comment]
@haardikdharma10 I agree, and I think using `warning` makes sense because this message is meant to alert the user to change something.
